src/pages/Account/Admin.tsx

```diff
@@ -207,6 +211,20 @@ const EventsComponent: React.FC<VenueDetailsPartProps> = ({ venue }) => {
                       <div className="price-container">
                         Individual tickets £{venueEvent.price / 100}
                       </div>
+                      <div className="event-payment-button-container">
+                        <div>
+                          <button
+                            role="link"
+                            className="btn btn-primary buy-tickets-button"
+                            onClick={() => {
+                              setShowCreateEventModal(true);
+                              setEditedEvent(venueEvent);
```

Comment:
why not just `setEditedEvent(venueEvent)` then do `{!!venueEvent && ..` for showing the modal

Reply:
Create uses the same modal, without an event